doc/doxygen/src/porting-boards.md

```diff
@@ -0,0 +1,294 @@
+Porting boards                                                 {#porting-boards}
+================
+
+At some point you might need to port a new `BOARD` to `RIOT`, either because
+that specific development board is not yet supported or because you have a
+custom `BOARD` for your project.
+
+If you want to port a `BOARD` to `RIOT` you have two choices: doing it
+inside of `RIOTBASE` or outside. In either case the file structure
+is basically the same and moving from one to another is easy.
+
+This guide details the generic structure you need to add a new `BOARD`
+to `RIOT`, the different files as well as their functionality.
+
+_note_: we assume here that your `CPU` and `CPU_MODEL` is already supported
+in `RIOT` so no peripheral or cpu implementation is needed.
+
+# General structure                                         {#general-structure}
+
+Like @ref creating-an-application "applications" or @ref creating-modules
+"modules", boards consist on a directory containing source files and
+makefiles. Usually a `BOARD` directory has the following structure
+
+```
+  board-foo/
+  |----dist/
+      |----scripts
+  |----board.c
+  |----doc.txt
+  |----include/
+      |----periph_conf.h
+      |----board.h
+      |----gpio_params.h
+  |----Makefile
+  |----Makefile.dep
+  |----Makefile.features
+  |----Makefile.include
+```
+
+## Source files                                            {#board-source-files}
+
+Header files in `board-foo/include` define physical mappings or
+configurations. e.g:
+
+- `periph_conf.h`: defines configurations and mappings for peripherals as well
+  as clock configurations.
+- `board.h`: holds board specific definitions or mappings, for examples leds,
+  buttons. It might as well override default drivers parameters (e.g.: assigning
+  specific pin connections to a lcd screen, radio, etc.). Some boards might also
+  define optimized `XTIMER_%` values (e.g. @ref `XTIMER_BACKOFF`).
```

Comment:
```suggestion
  define optimized `XTIMER_%` values (e.g. @ref XTIMER_BACKOFF).
```

This will fix the Travis failure (tested locally with the same doxygen version ;) )